contracts/upgradeable_contracts/stake_token_mediators/ForeignStakeTokenMediator.sol

```diff
@@ -0,0 +1,41 @@
+pragma solidity 0.4.24;
+
+import "../amb_erc677_to_erc677/ForeignAMBErc677ToErc677.sol";
+import "../../interfaces/IBurnableMintableERC677Token.sol";
+
+contract ForeignStakeTokenMediator is ForeignAMBErc677ToErc677 {
```

Comment:
Please consider to redefine `getBridgeMode` and `getBridgeInterfacesVersion` since the logic of parent contracts is changed. The same is for the home side.

Reply:
Addressed by a6d008d